docs/server-config.md

```diff
@@ -115,3 +119,37 @@ will be generated and displayed in the command line.
 
 To delete an account open password file in any text editor and delete the
 line with the user name.
+
+## Logging
+
+Logging for TabPy is implemented with standart Python logger and can be configured
+as explained in Python documentation at
+[Logging Configuration page](https://docs.python.org/3.6/library/logging.config.html).
+
+Default config proveded with TabPy is
+[`tabpy-server/tabpy_server/common/default.conf`](tabpy-server/tabpy_server/common/default.conf)
+and has configuration for console and file loggers. With changing the config
+user can modify log level, format of the logges messages and add or remove
+loggers.
+
+### Request Context Logging
+
+For extended logging (e.g. for auditing purposes) additional logging can be turned
+on with setting `TABPY_LOG_DETAILS` configuration file parameter to `true`.
+
+With the feature on additional information is logged for HTTP requests: caller ip,
```

Comment:
I think it would be useful to say specifically that the client is Tableau/Tableau Server to make users aware that this will log usernames from Tableau Server.

Reply:
Okay, I'll update the text together with the example below.